app/controllers/branches_controller.rb

```diff
@@ -1,24 +1,7 @@
 class BranchesController < ApplicationController
-  VISUALISATIONS = %w[list bubbles].freeze
-
   before_action :ensure_user_can_use_application!
   before_action :ensure_user_can_administer_taxonomy!, only: %i[edit_all update_all]
 
-  def show
-    @content_item = ContentItem.find!(params[:id])
-
-    @taxonomy_size = Taxonomy::TaxonomySizePresenter.new(
-      Taxonomy::TaxonomySize.new(@content_item)
-    )
-
-    respond_to do |format|
-      format.html
-      format.json do
-        render json: @taxonomy_size.nested_tree
-      end
-    end
-  end
-
   def edit_all
```

Comment:
(Comment slightly outside of the scope of this original PR...)

Perhaps this is an opportunity to clarify the `edit_all` and `update_all` actions too: when we're editing the branches, it is the single resource of the homepage. We're not _editing all_ the branches, but instead editing how branches are linked to a single content item. Perhaps one for another PR, but I just wanted to bring it up for discussion.

Reply:
I'm hoping that the work that Jos is doing as part of https://trello.com/c/TQgjro97/260-improve-the-management-of-level-1-taxons-in-content-tagger will remove these actions, and then we can potentially get rid of this entire controller, which would be good, as I agree that it's not very clear.